### PR TITLE
Added frontend to main screen

### PR DESCRIPTION
**Pull request by PhoeniX-art-lab**

### CODE REVIEW

File: frontend/templates/main.html [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
-  <a href="http://localhost:8080/register" class="btn btn-primary btn-block" type="button">Зарегистрироваться</a>
-</div>
-{% endblock %}
\ No newline at end of file
+
+    <div class="text-xl-center main-text h2">
+        <p align="center">Добро пожаловать в<br>Плантацию!</p>
+    </div>
+    <div class="d-grid gap-2 col-5 mx-auto my-auto">
+        <div style="margin-top: 157px">
```
Comment (Failjak) [inline frontend/templates/main.html:9]: I think using % is more flexible instead of px

File: frontend/templates/main.html [hunk 2]
```diff
@@ -1,8 +1,25 @@
...
+            <a href="http://localhost:8080/auth" class="start-buttons btn btn-primary btn-block" type="button">
+                <div class="text">
+                    Войти
+                </div>
+            </a>
+            <div style="margin-top: 40px">
```
Comment (Failjak) [inline frontend/templates/main.html:15]: same

File: frontend/templates/main.html (hunk 1 above)
Comment (PhoeniX-art-lab) [reply frontend/templates/main.html]: Changed this

File: frontend/templates/main.html (hunk 2 above)
Comment (PhoeniX-art-lab) [reply frontend/templates/main.html]: Changed this

**Pull request merged**